packages/boot/test/bootstrapTests/orchestration.test.ts

```diff
@@ -170,7 +175,9 @@ test.serial('stakeAtom - smart wallet', async t => {
   const { ATOM } = agoricNamesRemotes.brand;
   ATOM || Fail`ATOM missing from agoricNames`;
 
-  await wd.executeOffer({
+  // Cannot await executeOffer because the offer won't resolve until after the bridge's inbound queue is flushed.
+  // But this test doesn't require that.
```

Comment:
Should we require this test to check the offer result?

Reply:
that's covered in `restart-contracts`.

Reply:
Thanks, seeing that now. Maybe we can combine these files at some point in the future so it's easier to follow